### ISSUE TITLE
AttributeError: 'Connection' object has no attribute 'readonly'

### ISSUE THREAD

**Issue by lu828** [cloudflare/sqlalchemy-clickhouse#1]
I have met this issue: "AttributeError: 'Connection' object has no attribute 'readonly'" when query tables names. So I have analyzed your source. Based on my analysis of it, I have found one issue.
'Connection' class is inherited from Database but it does not inherit parent's method or attributes from Database class.
To fix it, I have added a following code.
```sh
super(Connection, self).__init__(db_name, db_url, username, password)
```
After that, it works for me well.

**Comment by vavrusa** [comment]
Merged #2 thanks!

**Comment by TheMcGoose** [comment]
@lu828 Do you have an example of how you're using this and it works? I'm currently trying the same thing and it's failing.

```python
import sqlalchemy as sa
engine = sa.create_engine("clickhouse://USERNAME:PASSWORD@HOSTNAME/DATABASE", readonly=True)
```

Stack Trace:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "python2.7/site-packages/sqlalchemy/engine/__init__.py", line 419, in create_engine
    return strategy.create(*args, **kwargs)
  File "python2.7/site-packages/sqlalchemy/engine/strategies.py", line 161, in create
    engineclass.__name__))
TypeError: Invalid argument(s) 'readonly' sent to create_engine(), using configuration ClickHouseDialect/QueuePool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
```

Version Information:
```
$ pip freeze
SQLAlchemy==1.2.2
sqlalchemy-clickhouse==0.1.3.post0
```
